src/applications/check-in/api/local-mock-api/mocks/v2/pre-check-in-data/get.js

```diff
@@ -115,6 +117,12 @@ const createMockSuccessResponse = (
     apptKind = 'phone';
     location = '';
   }
+  if (token === cvtApptUUID) {
```

Comment:
Could use a switch statement here since you've got three if statements checking one item now

Reply:
I'll think I'll leave this the way it is for now. The whole thing could probably be cleaned up a bit. There's also the sharedData get that does a lot of the same things.